packages/@lightningjs/ui-components/src/components/Tile/Tile.js

```diff
@@ -502,6 +502,16 @@ export default class Tile extends Surface {
     };
   }
 
+  get metadataLocation() {
+    return this.style.metadataLocation ?? this.metadataLocation;
+  }
+
+  set metadataLocation(newMetadataLocation) {
```

Comment:
I think this and the above should actually be `_setMetadataLocation` and `_getMetadataLocation` because those are the methods that the `withUpdates` mixin creates.

Reply:
Checkout out ProgressBar for an example of overriding the props from the properties array:
```js
  _setProgress(progress) {
    this._progressChanged = progress !== this._progress;
    return progress;
  }
```